src/lib/modify-vscode-editor.ts

```diff
@@ -0,0 +1,57 @@
+import * as vscode from 'vscode'
```

Comment:
This file deals with actually interacting with the vscode API to perform text modifications as required.